src/lib.rs

```diff
@@ -211,6 +225,13 @@ impl AsRef<[u8]> for Ciphertext {
 pub struct SharedSecret<'a>(KeyBuffer<'a, CRYPTO_BYTES>);
 
 impl<'a> SharedSecret<'a> {
+    /// Move the secret to the heap and make it `'static`
```

Comment:
Move is not really correct here right, this should be Copy?

Reply:
Yeah, true. I can work on a better wording there.